messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/AbstractMessagingDatabusConsumer.java

```diff
@@ -299,6 +295,15 @@ protected String getReaderNameSuffix() {
 
   protected abstract void createPartitionReaders() throws IOException;
 
+  public synchronized Long getPendingData() throws Exception {
```

Comment:
rename it to getPendingDataSize() to clearly convey that its returning pending data size and not count of pending rows.